src/web/test.rs

```diff
@@ -108,11 +120,12 @@ impl MockAdm {
     /// Return the passed in query params
     async fn params(&mut self) -> HashMap<String, String> {
         let query_string = self.request_rx.next().await.expect("No request_rx result");
-        Url::parse(&format!("{}{}", self.endpoint_url, query_string))
+        let bits = Url::parse(&format!("{}?{}", self.endpoint_url, query_string))
             .expect("Couldn't parse request_rx result")
             .query_pairs()
             .into_owned()
-            .collect()
+            .collect();
+        bits
```

Comment:
since we're no longer debugging this, let's just return 
```
Url::parse(&format!("{}?{}", self.endpoint_url, query_string))
            .expect("Couldn't parse request_rx result")
            .query_pairs()
            .into_owned()
            .collect()
```